docs/source/update_indexing.rst

```diff
@@ -0,0 +1,154 @@
+.. _update-indexing:
+
+******************************
+Update Indexing for cocotb 2.0
+******************************
+
+cocotb 2.0 changed what types were returned when getting a value from a
```

Comment:
```suggestion
cocotb 2.0 changes what types are returned when getting a value from a
```
?

Reply:
My reasoning for using present tense is that for users having a need to read this section, cocotb 2 is/will be "the new and current thing", regardless of when exactly they read it.